src/aria/utils/Array.js

```diff
@@ -173,6 +173,30 @@ var Aria = require("../Aria");
                     }
                 }
                 return output;
+            },
+            /**
+             * This tests whether all elements in the array pass the test implemented by the provided function. 
+             * As long as the elements return truthy values, the loop will continue. However, as soon as a falsy
+             * value is returns, the answer will be returned without analyzed the rest of the elements.
+             * @param {Array} array Array to filter.
+             * @param {Function} Function to test each element of the array. This function receive as arguments the
+             * value, the index and the array.
+             * @param {Object} thisObject Object to use as this when executing callback.
```

Comment:
Hi, I took the name thisObject from the rest of the methods in this utils class, would you suggest renaming all of the thisObjects (for the filter and forEach methods) to thisArgument?